routes/exchange.go

```diff
@@ -1617,41 +1618,63 @@ func (fes *APIServer) GetPostsForFollowFeedForPublicKey(bav *lib.UtxoView, start
 		return nil, errors.Wrapf(err, "GetPostsForFollowFeedForPublicKey: Problem filtering out restricted public keys: ")
 	}
 
-	minTimestampNanos := uint64(time.Now().UTC().AddDate(0, 0, -2).UnixNano()) // two days ago
-	// For each of these pub keys, get their posts, and load them into the view too
-	for _, followedPubKey := range filteredPubKeysMap {
+	var postEntriesForFollowFeed []*lib.PostEntry
+	n := 1
+	for n < 6 {
+		postEntriesForFollowFeed = nil
```

Comment:
so we are resetting this on each loop, and then re-querying the entire date range, until we reach the `numToFetch` value? why can't we keep adding onto this array until we reach the desired amount?